src/test/java/com/epam/test_generator/controllers/CaseControllerTest.java

```diff
@@ -331,64 +338,65 @@ public void addCase_RuntimeException_StatusInternalServerError() throws Exceptio
                 .content(mapper.writeValueAsString(caseDTO)))
             .andExpect(status().isInternalServerError());
 
-        verify(casesService).addCaseToSuit(anyLong(), anyLong(), any(CaseDTO.class));
+        verify(casesService).addCaseToSuit(anyLong(), anyLong(), any(CreateCaseDTO.class));
     }
 
     @Test
     public void updateCase_UpdateCase_StatusOk() throws Exception {
-        CaseUpdateDTO expectedDto = new CaseUpdateDTO(caseDTO, Arrays.asList(1L, 3L, 5L));
-        when(casesService.updateCase(SIMPLE_PROJECT_ID,SIMPLE_SUIT_ID, SIMPLE_CASE_ID, editCaseDTOList.get(0))).thenReturn(expectedDto);
+        CaseWithFailedStepsDTO expectedDto = new CaseWithFailedStepsDTO(caseDTO, Arrays.asList(1L, 3L, 5L));
+        when(casesService.updateCase(anyLong(), anyLong(), anyLong(), any(UpdateCaseDTO.class))).thenReturn(expectedDto);
+
+//        when(casesService.updateCase(SIMPLE_PROJECT_ID,SIMPLE_SUIT_ID, SIMPLE_CASE_ID, updateCaseDTO)).thenReturn(expectedDto);
```

Comment:
Do you have any reasons to save commented code?

Reply:
Fixed. Just a mistake.